cnv/cnv_users_guide/cnv-fencing-unhealthy-nodes.adoc

```diff
@@ -0,0 +1,27 @@
+[id="cnv-fencing-unhealthy-nodes"]
+= Fencing unhealthy nodes
+include::modules/cnv-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: cnv-fencing-unhealthy-nodes
+toc::[]
+
+Fencing is the process of isolating a failed node to protect a cluster and its resources.
+A node is considered 'fenced' when it can no longer access shared resources in order
+for those resources to be used elsewhere. Without fencing, a failed node can result in data corruption.
+
+In the case of an unhealthy node, Pods and virtual machines that depend on
+persistent storage cannot be replaced on another node until it can be confirmed
+that the original Pod or virtual machine is no longer accessing the storage.
+Fencing the node ensures that the workloads are no longer running on the
+unhealthy node, protecting the persistent storage. Once fencing is complete,
+workloads are safely replaced on other nodes.
```

Comment:
I think this sentence should be rephrased into the active voice. I am also not sure about "safely replaced" and wonder if there's a more accurate phrase. "Workloads resume"?